examples/integrations/oci/oci-secrets-mp-config-source/src/main/resources/mp-meta-config.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+add-default-sources: false
```

Comment:
Are these really required?  If not then lets remove it, otherwise it could use some commentary.

Reply:
Hmm; I feel like I did comment on this somewhere…maybe in the `README.md`. They are strictly speaking not needed but they help to constrain the example: the configuration sources will be, in other words, exactly what are listed under `sources` and no other. I'll copy and paste whatever pre-existing commentary I already did.

Reply:
Resolved in ~6e18584